packages/tc-api-s3-document-store/patch.js

```diff
@@ -28,10 +33,15 @@ const s3Patch = async ({ s3Instance, bucketName, type, code, body }) => {
 	}
 
 	const newBodyDocument = Object.assign(existingBody, body);
+
+	const newBodyWithoutEmpty = Object.fromEntries(
+		Object.entries(newBodyDocument).filter(([, value]) => !!value),
+	);
+
```

Comment:
Great. This was the piece of code I was looking for when I saw the bug.  Its much clearer that the new body should exclude blank/null/undefined fields.